### PR TITLE
Small fixes and improvements in tractogram filtering scripts

### PR DESCRIPTION
**Pull request by EmmaRenauld**
I worked a bit with tractogram filtering scripts this week, and here is a list of very small changes.

- My pycharm was sometimes confused by the datatypes. Clarifying docstrings.
- Adding --binary option to scil_compute_endpoints maps.
- Adding verbose option to scil_compute_qbx.
- Adding option to save rejected streamlines in scil_filter_tractogram

### CODE REVIEW

**Review by mdesco — APPROVED** [review]
@arnaudbore  can you do a quick check and merge this asap please.

File: scripts/scil_compute_qbx.py [hunk 1]
```diff
@@ -64,6 +66,10 @@ def main():
     clusters = qbx_and_merge(streamlines, thresholds,
                              nb_pts=args.nb_points, verbose=False)
 
+    if args.verbose:
+        print("Tractogram was separated into {} clusters. Saving..."
```
Comment (AntoineTheb) [inline scripts/scil_compute_qbx.py:70]: Utiliser le logger à la place ?

**Review by AntoineTheb — APPROVED** [review]
Have not tested. One small comment otherwise LGTM

**Comment by scil-admin** [comment]
Build passed ! Good Job :beers: !

**Comment by scil-admin** [comment]
Build passed ! Good Job :beers: !

**Review by arnaudbore — APPROVED** [review]

**Pull request merged**